web/src/designer/components/form-editor.tsx

```diff
@@ -74,6 +78,12 @@ export const FormEditor = ({
   const searchParams = new URLSearchParams(location.search);
   const sectionParam = searchParams.get('section');
 
+  const uiSpec = useAppSelector(
+    state => state.notebook['ui-specification'].present
+  );
+  // we need this to be a ProjectUIModel type for the PreviewFormManager
+  const uiSpecInternal: ProjectUIModel = {views: uiSpec.fviews, ...uiSpec};
```

Comment:
nit, but shouldn't you remove fviews from the resulting object? rather than spread it?